redbot/cogs/mod/settings.py

```diff
@@ -377,13 +377,22 @@ async def defaultdays(self, ctx: commands.Context, days: int = 0):
     async def defaultduration(
         self,
         ctx: commands.Context,
-        duration: Optional[commands.TimedeltaConverter] = timedelta(days=0),
+        *,
+        duration: commands.TimedeltaConverter,
     ):
-        """Set the default time to be used when a user is tempbanned."""
+        """Set the default time to be used when a user is tempbanned.
+
+        Accepts: seconds, minutes, hours, days, weeks
+        Use `0` with a unit of measure to set the time to zero.
```

Comment:
Should we allow to set 0 as the time? There's already softban for situations where you want to ban user and immediately unban them.
The converter allows setting the minimum duration, so this should be fairly easy and would actually simplify a few things (i.e. docstring, and the currently needed `or` in the `.format()`).